src/javascript/app/common/form_validation.js

```diff
@@ -314,7 +315,7 @@ const Validation = (() => {
         }
 
         let all_is_ok = true;
-        let message   = '';
+        const message = '';
```

Comment:
Do we still need this message if it is always going to be empty? 😅

Reply:
Removed this now